pom.xml

```diff
@@ -96,7 +96,7 @@
     <libcudf.build.configure>false</libcudf.build.configure>
     <libcudf.clean.skip>true</libcudf.clean.skip>
     <libcudf.install.path>${project.build.directory}/libcudf-install</libcudf.install.path>
-    <libcudf.dependency.mode>latest</libcudf.dependency.mode>
+    <libcudf.dependency.mode>pinned</libcudf.dependency.mode>
```

Comment:
Need we set to `pin` submodule-sync.sh below? 

https://github.com/NVIDIA/spark-rapids-jni/blob/branch-24.04/ci/submodule-sync.sh#L76

Reply:
No, the submodule sync job is the one we want to update to the latest dependencies.  All other builds should be using pinned dependencies.